### ISSUE TITLE
Когда обновление? 

### ISSUE THREAD

**Issue by swordily** [swordily/SPERMAPYTHON#2]
Я не хочу и не буду обновлять данный чит, у меня есть другие проекты по типу SPERMAWARE, SPERMASENSE, SPERMALOADER. На пайтоне я мало чего смогу сделать, чем на c++. Имеется все базовые функции и имеется авто обновление под ласт апдейт ксго, кто хочет добавить функции, качаете сурсы и добавляете, я не буду добавлять ничего.

**Comment by TxcClowan** [comment]
Он будет работать вечно да и что туда ещё добавить то можно? Всё есть. Играть на мэйне пойдёт. Но я так понимаю, что лучше играть с SPERMAWARE или с SPERMASENSE?

**Comment by swordily** [comment]
> Он будет работать вечно да и что туда ещё добавить то можно? Всё есть. Играть на мэйне пойдёт. Но я так понимаю, что лучше играть с SPERMAWARE или с SPERMASENSE?

Да, если хотите легит, то используйте спермаваре. Делается упор на легит, но рейдж тоже присутствует. SPERMASENSE сделан исключительно для рейджа, тестили против кряка, в4, фаталити и другими читами. Может убивать, так же есть дт в  1 тик, все 2 чита будут бесплатны, можно юзать SPERMALOADER чтобы не получить вак. Все читы тоже андетект

**Comment by TxcClowan** [comment]
> > Он будет работать вечно да и что туда ещё добавить то можно? Всё есть. Играть на мэйне пойдёт. Но я так понимаю, что лучше играть с SPERMAWARE или с SPERMASENSE?
> 
> Да, если хотите легит, то используйте спермаваре. Делается упор на легит, но рейдж тоже присутствует. SPERMASENSE сделан исключительно для рейджа, тестили против кряка, в4, фаталити и другими читами. Может убивать, так же есть дт в 1 тик, все 2 чита будут бесплатны, можно юзать SPERMALOADER чтобы не получить вак. Все читы тоже андетект

Почему не сделать рейдж и легит в один чит?

**Comment by swordily** [comment]
> > > Он будет работать вечно да и что туда ещё добавить то можно? Всё есть. Играть на мэйне пойдёт. Но я так понимаю, что лучше играть с SPERMAWARE или с SPERMASENSE?
> > 
> > 
> > Да, если хотите легит, то используйте спермаваре. Делается упор на легит, но рейдж тоже присутствует. SPERMASENSE сделан исключительно для рейджа, тестили против кряка, в4, фаталити и другими читами. Может убивать, так же есть дт в 1 тик, все 2 чита будут бесплатны, можно юзать SPERMALOADER чтобы не получить вак. Все читы тоже андетект
> 
> Почему не сделать рейдж и легит в один чит?

Потому что легит чит я сделал на другой базе, а рейдж на другой базе. И сейчас всё это переносить в один чит будет довольно трудно, тем более у всех разное меню, а не одно меню на 2 чита

**Comment by TxcClowan** [comment]
> > > > Он будет работать вечно да и что туда ещё добавить то можно? Всё есть. Играть на мэйне пойдёт. Но я так понимаю, что лучше играть с SPERMAWARE или с SPERMASENSE?
> > > 
> > > 
> > > Да, если хотите легит, то используйте спермаваре. Делается упор на легит, но рейдж тоже присутствует. SPERMASENSE сделан исключительно для рейджа, тестили против кряка, в4, фаталити и другими читами. Может убивать, так же есть дт в 1 тик, все 2 чита будут бесплатны, можно юзать SPERMALOADER чтобы не получить вак. Все читы тоже андетект
> > 
> > 
> > Почему не сделать рейдж и легит в один чит?
> 
> Потому что легит чит я сделал на другой базе, а рейдж на другой базе. И сейчас всё это переносить в один чит будет довольно трудно, тем более у всех разное меню, а не одно меню на 2 чита

Хорошо, спасибо. Буду ждать
